### PR TITLE
Update for removal of sdn image

### PR DESCRIPTION
**Pull request by danwinship**
openshift-sdn has been removed from the release image, update for that.

In 4.17, we are still building kube-proxy from the sdn repo, so this doesn't entirely remove the repo from the CI config.

(Yes, there is no CI that checks that the generated kube-proxy image actually works. This has always been the case...)

### CODE REVIEW

**Review by tssurya — APPROVED** [review]
lgtm

File: ci-operator/jobs/openshift/sdn/openshift-sdn-master-presubmits.yaml [hunk 1]
```diff
@@ -1,321 +1,5 @@
 presubmits:
   openshift/sdn:
```
Comment (tssurya) [inline ci-operator/jobs/openshift/sdn/openshift-sdn-master-presubmits.yaml:2]: when will this file go away? given there are no more jobs in sdn repo moving forward?

Comment (danwinship) [reply ci-operator/jobs/openshift/sdn/openshift-sdn-master-presubmits.yaml:2]: When we move the kube-proxy build somewhere else. (Or possibly just stop building it.)

**Pull request merged**